sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/SaveIntoDataSourceCommand.scala

```diff
@@ -46,4 +46,6 @@ case class SaveIntoDataSourceCommand(
 
     Seq.empty[Row]
   }
+
+  override def simpleString: String = s"SaveIntoDataSourceCommand ${dataSource}, ${mode}"
```

Comment:
https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/Utils.scala#L2631-L2638

Reuse `spark.redaction.regex`?

Reply:
I can use that, but I would need to expand the default for `spark.redaction.regex` as the `user` and `url` (some drivers allow credentials passed in the connection url) fields might also contain sensitive data. We should also change `JDBCRelation::toString` to include the redaction regex to be consistent.

I would argue not showing the jdbc properties at all because they provide little value and wrong redaction regex configuration could cause leaks to downstream log collection systems.

let me know if that makes sense, and I can modify this accordingly

Reply:
`SaveIntoDataSourceCommand ` is not being used for JDBC only. 

`JDBCRelation::toString` was already fixed in https://github.com/apache/spark/pull/15975 ?